kapitan/targets.py

```diff
@@ -314,6 +314,7 @@ def compile_target(target_obj, search_paths, compile_path, ref_controller, **kwa
     compile_objs = target_obj["compile"]
     ext_vars = target_obj["vars"]
     target_name = ext_vars["target"]
+    ref_controller.target_name = target_name
```

Comment:
This can probably be removed once you remove target_name from RefController as well.